src/System.Text.Json/src/System/Text/Json/Linq/JNode.cs

```diff
@@ -5,8 +5,10 @@
 namespace System.Text.Json.Linq
 {
     /// <summary>
-    ///   The base class that represents a single node within a mutable JSON document.
+    ///   The base class that represents a single node within a mutable JSON document. See also <see cref="JObject"/> and <see cref="JArray"/> which derive from <see cref="JNode"/>.
```

Comment:
If we add references to `JNode` in `JsonDocument` documentation, maybe it would be worth mentioning `JsonDocument` in `JNode` documentation as well?